generate_assets_metadata.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
```

Comment:
since we have more and more scripts in the repo root, should we move them into a dedicated directory in another pr?

Reply:
sounds good 👍
I will create a task for that